docs/server-side/v2/active-rooms/get-peers.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
+nav: 3.9
+---
+
+You can use this API for get the active peers in the room. Peers can be filtered by specifying user_id or role in the query params
+
+<PostRequest title="https://api.100ms.live/v2/active-rooms/<room_id>/peers" />
+<Request id="req-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/active-rooms/<room_id>/peers' \
+--header 'Authorization: Bearer <management_token>
+```
+
+</Request>
+<ResponseBox id="res-0" status="200 OK">
+
+```json
+{
+    "peers": {
+        "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20": {
+            "id": "dcd88eb0-83e6-486a-bcae-c3be5e0f2c20",
+            "name": "John",
+            "user_id": "da3136fd-622e-468f-96ca-09c8c9f50072",
+            "metadata": "",
+            "role": "host",
+            "joined_at": "2023-01-25T12:06:34.080990401Z"
+        },
+        "4e03dce3-ddec-4bdd-b91b-0b18be1a7f10": {
+            "id": "4e03dce3-ddec-4bdd-b91b-0b18be1a7f10",
+            "name": "Nick",
+            "user_id": "8f12c699-53f2-471e-9397-9f937de2f8e7",
+            "metadata": "",
+            "role": "host",
+            "joined_at": "2023-01-25T12:04:30.366388779Z"
+        }
+    }
+}
+```
+
+</ResponseBox>
+
+## Allowed filters
+
+| Name          | Type      | Description                                                       | Required |
+| :------------ | :-------- | :-----------------------------------------------------------------| :------- |
+| user_id <br/> | `string`  | Determines the peers with specified user_id                       | No       |
```

Comment:
shall we add some info about how the user_id is generated. 
refer to [this](https://www.100ms.live/docs/server-side/v2/Sessions/example-build-attendance#calculate-the-individual-participants-duration).

Reply:
I can add this (your internal user identifier used while creating the [auth token](https://www.100ms.live/docs/server-side/v2/introduction/authentication-and-tokens#app-token) for peers).